packages/formatters-util/src/util/parseNumber.ts

```diff
@@ -1,18 +1,47 @@
+const replaceAllButLastOccurence = (input: string | string[], search: string) => {
+    // indexOf returns the first index, so flip the input to find the
+    // notator furthest back
+    const arrInput = Array.isArray(input) ? input.reverse() : input.split("").reverse();
+    const indexOfSearch = arrInput.indexOf(search);
+
+    return (
+        arrInput
+            // remove all instances of the notator except the last one
+            .filter((f, i) => {
+                if (f !== search) {
+                    return true;
+                }
+
+                return i === indexOfSearch;
+            })
+            // flip back and return as string
+            .reverse()
+            .join("")
+    );
+};
+
 export function parseNumber(input: string | number) {
     if (typeof input === "number") {
         return input;
     }
 
-    // fjern tegn som ikke kan tolkes som tall og erstatt kommadesimal
-    const strippedInput = input
-        .replace(/[^\d.,-]/g, "")
-        .replace(/[.,](\d{3})/g, "$1")
-        .replace(/,/g, ".");
-    const parsedNumber = Number.parseFloat(strippedInput);
+    // remove all spaces from number
+    const arrNumber = input.split("").filter((n) => n !== " ");
 
-    if (Number.isNaN(parsedNumber) || typeof parsedNumber !== "number") {
-        return null;
-    }
+    // find what separator is used for decimal notation
+    const decimalNotator = arrNumber.reduce<"." | "," | null>((notator, currentItem) => {
```

Comment:
Hva skjer hvis jeg har pølsefingre og skriver `1,200.300.001,31`? 

Sluttmålet her er å få en string med bare én forekomst (den siste, desimalskilleren) av `.` så Number ikke hyler, sant?

Pseudokodetid:

- `replaceAll(",", ".")`
- `removeAllButLastOccurence(thing, ".")`
- `return Number(fromResult)`

Reply:
Min tanke er at det er så langt unna et tall at om man har noe sånt i sine systemer har man større problemer og bør ikke få det gjennom.

Brukere bør heller ikke få lov til å fylle inn tall med tusennotasjon.

Med andre ord har tanken min vært å legge til rette for at Number ikke skal kaste opp helt av tall, men ikke slippe igjennom hva som helst.